geodesic_apkindex.md5

```diff
@@ -1 +0,0 @@
-77e3cde7afac220dbf00e433fbaf4abb
```

Comment:
You should not have an empty `geodesic_apkindex.md5`. Do you maybe have `*.md5` in your global `.gitignore`?

Reply:
I ran the update command but didn’t pay attention to the result. Will check what’s wrong next time.